e2e/libs/node/aws.py

```diff
@@ -0,0 +1,62 @@
+import boto3
+import logging
+
+class EC2:
```

Comment:
https://litmuschaos.github.io/litmus/experiments/categories/nodes/node-restart/#reboot-command Experiments can be done to investigate the feasibility. If so, we can get rid of this class.

Reply:
I think it is a way to improve and make the code structure more clearly and easily maintain in the future. I will create a ticket to follow and investigate later.

Reply:
I've create a ticket: https://github.com/longhorn/longhorn/issues/6499 for tracking and investigate next.